pages/docs/platform/monitor/insights.mdx

```diff
@@ -0,0 +1,209 @@
+import { Info } from "src/shared/Docs/mdx";
+
+export const description = 'Query and analyze event data with SQL in the Inngest platform.';
+
+# Insights
+
+Inngest Insights allows you to query and analyze your event data using SQL directly within the Inngest platform. Every event sent to Inngest contains valuable information, and Insights gives you the power to extract meaningful patterns and analytics from that data.
+
+<Info>
+  Insights support is currently in Public Beta. Some details including SQL syntax and feature availability are still subject to change during this period. Read more about the Public Beta [roadmap here](#roadmap).
+</Info>
+
+## Overview
+
+Insights provides an in-app SQL editor and query interface where you can:
+- Query event data using familiar SQL syntax
+- Save and reuse common queries
+- Analyze patterns in your event streams
+- Extract business intelligence from your workflows
+
+Currently, you can **only query events**. Support for querying function runs will be added in future releases.
+
+## Getting Started
+
+Access Insights through the Inngest dashboard by clicking on the "Insights" tab in the left navigation. 
+
+![Getting Started Dashboard View](/assets/docs/platform/monitor/insights/insights_dashboard_view.png)
+
+We have several pre-built query templates to help you get started exploring your data.
+
+![Getting Started Templates View](/assets/docs/platform/monitor/insights/insights_template_view.png)
+
+## SQL Editor
+
+The Insights interface includes a full-featured SQL editor where you can:
+
+- Write and execute SQL queries against your event data
+- Save frequently used queries for later access
+- View query results in an organized table format
+- Access query history and templates from the sidebar
+
+![Sql Editor View](/assets/docs/platform/monitor/insights/insights_sql_editor.png)
+
+### Available Columns
+
+When querying events, you have access to the following columns:
+
+| Column | Type | Description |
+|--------|------|-------------|
+| `event_id` | String | Unique identifier for the event |
+| `event_name` | String | The name/type of the event |
+| **`event_data`** | **JSON** | **The event payload data - users can send any JSON structure here** |
+| `event_ts` | DateTime | Timestamp when the event occurred |
```

Comment:
this is a unix millis timestamp. Not sure what the best reference to link would be, maybe https://en.wikipedia.org/wiki/Unix_time or https://www.unixtimestamp.com/. Usually unix timestamps are in seconds so * 1000 to get “unix millis”